src/loader/index.js

```diff
@@ -1,4 +1,7 @@
 import _ from "lodash";
+
```

Comment:
Refactored this to have a loader which can load secrets or load from environment. Combing happens a level up in `nodule`.